File: 8-kyu/How many stairs will Suzuki climb in 20 years?.py
from functools import reduce

def stairs_in_20(stairs):
    result = 0;
    for i in stairs:
         result += reduce(lambda a,b : a + b, i)
    return result * 20

#clever solution
def stairs_in_20(stairs):
    return 20 * sum(map(sum, stairs))

File: 8-kyu/How many stairs will Suzuki climb in 20 years?.js
function stairsIn20(s){
  let result = 0;
  for (let i = 0; i < s.length; i += 1) {
    result += s[i].reduce((acc,e)=>acc+e,0)
  }
  return result * 20
}

//clever soluiton
function stairsIn20(a) {
  return 20 * a.reduce((s, a) => s + a.reduce((s, n) => s + n, 0), 0);
}
